README.md

```diff
@@ -1,11 +1,57 @@
-Rokka PHP Client
-================
+# Rokka PHP Client
 
-Access library to the [Rokka.io image service](https://rokka.io).
+A [PHP](http://php.net/) library to access the API of the [Rokka](https://rokka.io/) image service.
 
-Most current documentation can always be found on the [Rokka homepage](https://rokka.io/documentation).
+If you are using the Symfony framework, have a look at the [Rokka Symfony Bundle](https://raw.githubusercontent.com/rokka-io/rokka-client-bundle) which integrates this library into Symfony.
 
-Running tests
--------------
+## Installation
 
-Run `vendor/bin/phpunit` from project root
+Require the library using composer:
+
+`composer require rokka/client`
+
+## Bootstrapping
+
+The `Rokka\Client\Factory` is the entry point for creating the API client.
+
+You will need to create an account on Rokka to get your api key and secret and create an organization.
```

Comment:
In this sprint the organization will be created automatically when you sign up. So maybe the best line here would be "You will need to register for a Rokka account and use the api key and secret that you recieved" it doesn't have to mention the "create an organization" step.